net-misc/gns3-server/gns3-server-2.2.22.ebuild

```diff
@@ -41,7 +48,9 @@ src_prepare() {
 	rm gns3server/compute/docker/resources/bin/busybox || die
 
 	# Package installs 'tests' package which is forbidden
-	rm -r tests || die
+	if ! use test; then
+		rm -r tests || die
+	fi
```

Comment:
Did you try to install this with tests enabled? It brings back the original issue:
```
 * ERROR: net-misc/gns3-server-2.2.22::gentoo failed (install phase):
 *   Package installs 'tests' package which is forbidden and likely a bug in the build system.
```
Not familiar with python stuff much, but preferably I believe should fix the setup.py, and get that upstreamed so can remove that later.
Similar to https://github.com/catkin/catkin_tools/pull/499/commits/2b95f71b9830e58003b52dff1a6be01390096c5a if that can help.

Reply:
Sorry, my bad. I've only tried it with `ebuild ... test/install`

I've looked into it and was a bit baffled actually.
What was surprising for me is that, while `gns3-server` does indeed try to install the tests directory, `gns3-gui` on the other hand does not! Both packages do have an identical directory structure and both even have the `tests` directory excluded in their `setup.py` configuration. - _so why did one package install tests but not the other_

First i was thinking emerge/portage/eclass has an issue here, but since this doesn't happened only on gentoo i noticed that whenever there exits a `__init__.py` file in a `tests` directory, the exclude wouldn't work until the exclude looks like `tests*` (as you pointed out - thanks again)

I don't know why it behaves like this and i also couldn't find any reasoning after a quick search, unfortunetaly my python knowledge is also very limited. 

Anyway, i've added another `sed` to fix it temporally and opened an PR upstream with the proposed fix.